rail_scripts/rail-preprocess-parquet

```diff
@@ -83,7 +83,8 @@ def parse_cmdline():
                          rows=rows, flux=args['--flux'],
                          output_template=args['--output-template'],
                          ra=args['--ra'], 
-                         dec=args['--dec'])
+                         dec=args['--dec'],
+                         objectId=args['--objectId'])
```

Comment:
Inconsistent coding style: please use hyphen notation for command line arguments: `--object-id`.